src/base/compute/owl_computation_symbol_sig.ml

```diff
@@ -89,35 +125,41 @@ module type Sig = sig
   val set_reuse : attr Owl_graph.node -> bool -> unit
   (** TODO *)
 
-  val get_reuse : attr Owl_graph.node -> bool
-  (** TODO *)
-
-  val set_vnode : attr Owl_graph.node -> t array -> unit
+  val is_reusable : attr Owl_graph.node -> bool
```

Comment:
Can we change this back to `get_reuse` if you dont have strong objection? Just for being consistent with other getter and setter functions.